src/services/api/apiAuthService.js

```diff
@@ -0,0 +1,20 @@
+import { defaultClient } from "./client";
+
+export const register = async (credentials) =>
+  defaultClient.post("/register", {
+    user: {
+      email: credentials.email,
+      password: credentials.password,
+      date_of_birth: "1990-01-01",
```

Comment:
at some point you'll probably want to set this value to some function argument, but for now it's fine 👍

Reply:
I will add this to my todo list for registration section, thank you! 👍🏽